pkg/backup/client.go

```diff
@@ -778,6 +778,11 @@ func OnBackupResponse(
 		log.Error("backup occur cluster ID error", zap.Reflect("error", v), zap.Uint64("storeID", storeID))
 		return nil, 0, errors.Annotatef(berrors.ErrKVClusterIDMismatch, "%v on storeID: %d", resp.Error, storeID)
 	default:
+		// UNSAFE! TODO: use meaningful error code instead of unstructured message to find failed to write error.
+		if messageIsRetryableFailedToWrite(resp.GetError().GetMsg()) {
+			log.Warn("backup occur failed to write file error.", zap.String("error", resp.GetError().GetMsg()))
```

Comment:
what about change the warn log to `backup occur s3 storage error`?

Reply:
OK